code/game/objects/effects/spawners/random/techstorage.dm

```diff
@@ -0,0 +1,117 @@
+// Tech storage circuit board spawners
+/obj/effect/spawner/random/techstorage
```

Comment:
Since these are under the 'random' type path, it's not immediately obvious that these spawners create their entire item lists without going to the code. Could you make this clearer from the object name? My suggestion would be a suffix like:
```
/obj/effect/spawner/random/techstorage/rnd_all 
```

Reply:
Done with https://github.com/tgstation/tgstation/pull/60522/commits/961bac541339366aab39f29e0a9a50fb68c0ccfa